configure

```diff
@@ -130,6 +130,8 @@ DSC_SCRIPT_PATH=$DSC_SCRIPT_PATH
 
 BUILD_DEBUG=$BUILD_DEBUG
 DEBUG_FLAGS=$DEBUG_FLAGS
+# DSC cannot build with -Werror
+ENABLE_WERROR=0
```

Comment:
Agreed for the override.
I was referring to the [top level makefile](https://github.com/Microsoft/PowerShell-DSC-for-Linux/blob/develop/Makefile#L2)